CDP4Composition/Services/RuleVerification/RuleVerificationService.cs

```diff
@@ -9,6 +9,8 @@ namespace CDP4Composition.Services
     using System;
     using System.Collections.Generic;
     using System.ComponentModel.Composition;
+    using System.Reactive.Linq;
```

Comment:
when you update a source-file, feel free to update the copyright header with the one referencing GPL